src/main/java/com/google/impactdashboard/data/recommendation/Recommendation.java

```diff
@@ -24,10 +25,10 @@
    * and accepted by the user at time {@code timestamp}, with description 
    * {@code description} and additional metadata {@code metadata}.
    */
-  public static Recommendation create(String projectId, String actor, 
+  public static Recommendation create(String projectId, String organizationId, String actor, 
```

Comment:
What if a project doesn't have an organization?

Reply:
We've decided to store those as organizationId="N/A" and organizationName="N/A"